games-arcade/rocksndiamonds/patches/rocksndiamonds-3.3.1.2.patchset

```diff
@@ -1,4 +1,4 @@
-From ba88fe1ad26fc936fcfece46929b4b1eebb091c1 Mon Sep 17 00:00:00 2001
+From ba88fe1ad26fc936fcfece46929b2b1eebb091c1 Mon Sep 17 00:00:00 2001
```

Comment:
Cool: I noticed that you are taking care of keeping the previous SHA1 checksums. Thanks!

(There is only one left: you can replace "ba88fe1ad26fc936fcfece46929b2b1eebb091c1" by "ba88fe1ad26fc936fcfece46929b4b1eebb091c1")